### PR TITLE
Update hci_helper.R

### PR DESCRIPTION
**Pull request by AaronPlex**
I updated the color palette for each produced map.

### CODE REVIEW

**Review by krishnakeshav — APPROVED** [review]
You are a developer now

**Review by AaronPlex — COMMENTED** [review]
@krishnakeshav

File: docs/articles/analysis_files/figure-html/non-global-3.png
Comment (krishnakeshav) [inline docs/articles/analysis_files/figure-html/non-global-3.png:1]: Verifying new palette @AaronPlex

File: docs/articles/analysis_files/figure-html/final-3.png
Comment (krishnakeshav) [inline docs/articles/analysis_files/figure-html/final-3.png:1]: Looks good @AaronPlex

**Review by krishnakeshav — APPROVED** [review]

**Review by krishnakeshav — COMMENTED** [review]
@AaronPlex Sure! Do indicate when all the changes are complete. I will compile the changes.

File: geohabnet/R/params.R [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-#' Retrieves the parameters and copies the parameter file to the specified
+#' This function retrieves the parameters and copies the parameter file to the specified
 #' output path.
 #' @param out_path character. The output path where the parameter file will be
-#' copied. Default is temporary directory [tempdir()]
-#' @param iwindow logical. If `TRUE`, prompts the user to select the output
-#' directory using a file chooser window. Default is `FALSE`
+#' copied. The default is a temporary directory [tempdir()]
+#' @param iwindow logical. If window = `TRUE`, this will prompt the user to select the output
+#' directory using a file chooser window. The default parameter is set to `FALSE`.
```
Comment (krishnakeshav) [inline geohabnet/R/params.R:9]:
```suggestion
#' directory using a file chooser window. The default value is `FALSE`.
```
`_parameter_` is a reference, _default_ is a value.

File: geohabnet/R/params.R [hunk 2]
```diff
@@ -80,9 +80,9 @@ load_parameters <- function(filepath = .param_fp()) {
 
 #' Get resolution value
 #'
-#' Resolution stored in `parameter.yaml`.
-#' If not present it will result default value.
-#' @return Numeric. Resolution from `parameters.yaml`. Default is 24.
+#' Resolution stored in `parameter.yaml`. Here, resolution values refer to the aggregation factor or granularity. Granularity is the number of small grid cells that are aggregated into larger grid cells in each direction (horizontally and vertically). For example, the finest spatial resolution of the Monfreda and MAPSPAM datasets in geohabnet is 5 minutes, a granularity value of 6 will result in maps with a spatial resolution of 0.5 degrees.
+#' If not provided, the resolution value used for the analysis is by default 24 (or two degrees when using the Monfreda and MAPSPAM datasets). Otherwise, a single integer value for granularity equal to or greater than one should be specified.
+#' @return Numeric. Resolution from `parameters.yaml`. The default is 24.
```
Comment (krishnakeshav) [inline geohabnet/R/params.R:85]:
```suggestion
#' @return Numeric. Resolution from `parameters.yaml`. The default is 12.
```

**Review by krishnakeshav — COMMENTED** [review]
looks good! @AaronPlex

**Pull request merged**